elements/order_slip.php

```diff
@@ -123,11 +123,11 @@
         </p>
 
         <p>
-            <strong><?= t("Payment Method") ?>: </strong><?= $order->getPaymentMethodName() ?><br>
+            <strong><?= t("Payment Method") ?>: </strong><?= t($order->getPaymentMethodName()) ?><br>
```

Comment:
No, I think I'm wrong here. It does need to be translated.
I'm mixing it up with the Display Name for a payment method.

However it probably should be the Display Name here, so it matches what a customer would have seen in the checkout. (don't worry about that now)